File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/interfaces/interfaceClassWithInterfaceFieldsOfDifferentType/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.interfaces.interfaceClassWithInterfaceFieldsOfDifferentType.expected;

import java.lang.String;

public class DgsConstants {
  public static class VEGETARIAN {
    public static final String TYPE_NAME = "Vegetarian";

    public static final String Calories = "calories";

    public static final String Vegetables = "vegetables";
  }

  public static class DOG {
    public static final String TYPE_NAME = "Dog";

    public static final String Name = "name";

    public static final String Diet = "diet";
  }

  public static class PET {
    public static final String TYPE_NAME = "Pet";

    public static final String Name = "name";

    public static final String Diet = "diet";
  }

  public static class DIET {
    public static final String TYPE_NAME = "Diet";

    public static final String Calories = "calories";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/interfaceClassWithInterfaceFieldsOfDifferentType/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.interfaceClassWithInterfaceFieldsOfDifferentType.expected

import kotlin.String

@Generated
public object DgsConstants {
  @Generated
  public object VEGETARIAN {
    public const val TYPE_NAME: String = "Vegetarian"

    public const val Calories: String = "calories"

    public const val Vegetables: String = "vegetables"
  }

  @Generated
  public object DOG {
    public const val TYPE_NAME: String = "Dog"

    public const val Name: String = "name"

    public const val Diet: String = "diet"
  }

  @Generated
  public object PET {
    public const val TYPE_NAME: String = "Pet"

    public const val Name: String = "name"

    public const val Diet: String = "diet"
  }

  @Generated
  public object DIET {
    public const val TYPE_NAME: String = "Diet"

    public const val Calories: String = "calories"
  }
}
